BBBread.py

```diff
@@ -256,16 +346,20 @@ def __init__(self, path=CONFIG_PATH, remote_host_1=LA_SERVER_IP, remote_host_2=C
         self.listen_thread.start()
         self.logger.debug("Listening thread started")
         self.logger.debug("BBBread startup completed")
+        self.logs_name = "BBB:{}:{}:Logs".format(self.bbb_ip, self.bbb_hostname)
+        self.server_connected = False
```

Comment:
This blocks 'pinging_thread' from updating db every 10 seconds!
Should move to line 322 instead?

Reply:
Updated!